utilities/mtv_migration.py

```
@@ -373,6 +373,52 @@ def wait_for_migration_complate(plan: Plan) -> None:
+                    break
+                LOGGER.debug(
+                    f"Waiting for VM '{vm_name}' NIC data in inventory... "
+                    f"(NICs: {vm.get('nics', []) if vm else 'VM not found'})"
+                )
+        except TimeoutExpiredError:
+            vm_data = source_provider_inventory.get_vm(name=vm_name)
+            nics = vm_data.get("nics", []) if vm_data else []
+            raise TimeoutExpiredError(
+                f"Timed Out: VM '{vm_name}' found in Forklift inventory but NIC/network data "
+                f"did not sync after {timeout}s. NICs: {nics}"
+            )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)